### DISCUSSION TITLE
Can this be used for any project?

### DISCUSSION THREAD

**Discussion by smsajid** [cncf/landscapeapp#895] [Q&A] [answered]
I want to use this project to build a landscape for my project. My project is not related to Linux Foundation. 
I have concerns regarding:
- how to merge updates from this project?
- how to have my own build infrastructure?

**Comment by nate-double-u** [comment, marked as answer]
Hi @smsajid, the [license used is Apache 2.0](https://github.com/cncf/landscapeapp?tab=Apache-2.0-1-ov-file#readme). You'd have to decide if that's approprite for the project you'd like to use the landscape for, but it's a very permissive license.

> how to merge updates from this project?

There are a lot of ways [to merge from upstream](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork), you can also search for "syncing a fork" to find information about this.

> how to have my own build infrastructure?

Check out the [Creating a new landscape](https://github.com/cncf/landscapeapp?tab=readme-ov-file#creating-a-new-landscape) section of the readme. There's a lot of information about the requirements and how to go about making your own.
